lib/selbot2/cla.rb

```diff
@@ -0,0 +1,27 @@
+require 'uri'
+
+module Selbot2
+  class CLA
+    include Cinch::Plugin
+
+    prefix Selbot2::PREFIX
+    match /(?:cla) (.+)/
+
+
+    def execute(message, query)
+      commits_url = "https://github.com/SeleniumHQ/selenium/commits?author=#{URI.escape query}"
+      resp   = RestClient.get(commits_url)
+      result = Nokogiri.XML(resp)
+
+      if result.css('.commit').any?
+        message.reply "#{query} has signed the CLA and has #{result.css('.commit').size} commits."
```

Comment:
I'm hoping that I somehow scrounge up a few hours of time to write an app that will read the cla spreadsheet for us, that won't leak any sensitive information and that simon runs (since he's the only one with access).

Then we'd setup a Pull Request bot of some kind that will validate the user has signed it and mark it in github accordingly.

But I don't mind this being called 'cla' for selbot, just not give a misconception that we've somehow checked the cla was signed.  (imagine someone said they signed it, but didn't really and we merged their commit... :) )